Tests/FluentAssertions.Specs/Numeric/ComparableSpecs.cs

```diff
@@ -21,20 +21,18 @@ public void When_two_instances_are_equal_it_should_succeed()
         }
 
         [Fact]
-        public void When_two_instances_are_the_same_reference_but_are_not_considered_equal_it_should_not_succeed()
+        public void When_two_instances_are_the_same_reference_but_are_not_considered_equal_it_should_succeed()
```

Comment:
🤔 I can't get my head around this one ;-)

Reply:
Yeah it's quite unlikely that someone would implement a type that is not reflexive.